tests/local_config.py

```diff
@@ -0,0 +1,10 @@
+# This file contains secrets used by the tests
+
+from mara_db import dbs
+
+# supported placeholders
+#   host='DOCKER_IP' will be replaced with the ip address given from pytest-docker
+#   port=-1 will be replaced with the ip address given from pytest-docker
+
+POSTGRES_DB = dbs.PostgreSQLDB(host='DOCKER_IP', port=-1, user="mara", password="mara", database="mara")
```

Comment:
Why not pick that from an env var and add a fixture to set the env var?

Reply:
Yee... I am not 100% sure about that. This is actually a copy from the test suite of the `mara_db` project. See [local_config.py.example](https://github.com/mara/mara-db/blob/main/tests/local_config.py.example).

The main purpose that I defined it in a separate config file is that you have the option to simply activate / disable the tests for specific database engines. Especially for test against cloud services this is handy. I don't want to share the credentials for my cloud with everybody but at the same time share the option for those who want to test their changes against the cloud ;-)

Removing this and integrating it into the fixture makes sence for now, but maybe not in the future...